README.md

```diff
@@ -207,17 +207,24 @@ Here are the available configurations:
 
 ## Directive
 
-You can use the `[dpDayPicker]` directive to attach the picker to any component with an `ngModel`.
+You can use the `[dpDayPicker]` directive to attach the picker to any component with an `ngModel` or a `FormControl` (using reactive forms).
 
 i.e.
 ```html
-<input name="someName" [(ngModel)]="selectedDate" [dpDayPicker]="config"></input>
+<input name="someName" [(ngModel)]="selectedDate" [dpDayPicker]="config" />
+```
+
+or using reactive forms:
+```html
+<input name="someName" formControlName="selectedDate" [dpDayPicker]="config" />
```

Comment:
`[formControl]` is for a stand alone `FormControl` object.  `[formControlName]` is for referencing a `FormControl` that is part of a `FormGroup` object.  Both need to be supported, but it's easy to support them both.  I just inject the `NgControl` and angular takes care of figuring out where to get it from.